app/services/schools/generate_analysis_chart_configuration.rb

```diff
@@ -6,7 +6,7 @@ def initialize(
         school,
         aggregated_meter_collection,
         fuel_configuration,
-        dashboard_fuel_types = DashboardConfiguration::DASHBOARD_FUEL_TYPES,
+        dashboard_fuel_types = {},
```

Comment:
Not entirely sure what this does but it looks like quite a few things depend on configuration returned later on? Was it intended?